### DISCUSSION TITLE
Nebula connection status info

### DISCUSSION THREAD

**Discussion by saysairam** [slackhq/nebula#667] [Q&A]
Hi Team
   
Need some info or guidelines to know the Nebula tunnel/connection status to lighthouse. Is there any options to know the lighthouse connectivity from Linux Nebula client interface/OS platform ? For example, link state - UP , running etc.. Also suggest any other mechanism to know the lighthouse connection from platform side.

Thanks,
Sai

**Comment by SamSirry** [comment]
`ping` is your long-time friend.

**Comment by johnmaguire** [comment]
I recommend enabling the built-in SSH debug server: https://nebula.defined.net/docs/config/sshd/

You may find relevant info in this ticket: https://github.com/slackhq/nebula/issues/545

For a full list of commands, you'll have to go to the source (sorry!): https://github.com/slackhq/nebula/blob/8822f1366c1111feb2f64fef229eed2024512104/ssh.go#L176
